app/assets/stylesheets/forever_style_guide/modules/_modals.scss

```diff
@@ -0,0 +1,57 @@
+// preventing  .style-guide-partial .style-guide-partial-rendered > * from forcing modal to render inline 
+.style-guide-partial .style-guide-partial-rendered .modal {
+  position: fixed;
+}
+
+.modal-dialog {
+  @media (min-width: $screen-md-min) {
+    width: 720px;
+  }
+}
+.modal-content {
+  border: 0;
+  border-radius: $border-radius-large;
+  @include box-shadow(transparentize($color-black, 0.6) 0 0 12px);
+}
+.modal-header {
+  @include center_with_flexbox(center, center);
+  background-color: color('gray-300');
+  border-top-right-radius: $border-radius-large;
+  border-top-left-radius: $border-radius-large;
+
+  .close {
+    -webkit-flex: 1 0 0;
```

Comment:
Two questions:

1 - considering cross browser limitations of IE9 for flexbox how do we feel about this? http://caniuse.com/#feat=flexbox

2 - Compass provides flexbox mixin for handling browser prefixes: http://compass-style.org/examples/compass/css3/flexbox/